test/quantization/test_quant_api.py

```diff
@@ -526,6 +526,31 @@ def test_quantized_tensor_subclass_8da4w(self):
         self.assertTrue(torch.equal(res, ref))
 
     @unittest.skipIf(not TORCH_VERSION_AT_LEAST_2_4, "Test only enabled for 2.4+")
+    def test_quantized_tensor_subclass_8da4w_no_clipping_err(self):
```

Comment:
maybe this can be merged with the previous test, using https://github.com/pytorch/ao/blob/e1039abac7f429a8d7f489d047d9b34d6ac6afe2/test/dtypes/test_affine_quantized_float.py#L52